src/sentry/api/helpers/group_index/update.py

```diff
@@ -766,6 +778,17 @@ def handle_has_seen(
         GroupSeen.objects.filter(group__in=group_ids, user_id=user_id).delete()
 
 
+def handle_priority(priority: str, reason: str, group_list: Sequence[Group], actor: User) -> None:
+    for group in group_list:
+        update_priority(
+            group=group,
+            priority=PRIORITY_STR_TO_LEVEL_ENUM[priority] if priority else None,
+            reason=PRIORITY_CHANGE_REASON_STR_TO_ENUM[reason] if reason else None,
+            actor=actor,
+        )
+        group.update(priority_locked_at=django_timezone.now())
```

Comment:
Should `priority_locked_at` be accessible by get? @malwilley

Reply:
Do we need this to be returned in the response from get to ensure the value was updated?

Reply:
It's not being used in the UI so not necessary to include, but let's just do it anyway in case we want it in the future